setup.cfg

```
@@ -1,59 +1,65 @@
-[build_sphinx]
-source-dir = docs
-build-dir = docs/_build
-all_files = 1
-
-[build_docs]
-source-dir = docs
-build-dir = docs/_build
-all_files = 1
-
-[upload_docs]
-upload-dir = docs/_build/html
-show-response = 1
-
 [tools:pytest]
-minversion = 2.2
-norecursedirs = build docs/_build
-remote_data_strict = true
-
-[ah_bootstrap]
-auto_use = True
+testpaths = "pypi" "docs"
+astropy_header = true
+doctest_plus = enabled
+text_file_format = rst
+addopts = --doctest-rst
 
 [flake8]
 max-line-length = 110
 max-doc-length = 79
-exclude = __init__.py, setup_package.py, conf.py, setup.py, ah_bootstrap.py, astropy_helpers, version.py, ci-helpers conftest.py
+exclude = __init__.py, setup_package.py, conf.py, setup.py, version.py, ci-helpers conftest.py
```

not sure what ci-helpers doing here

```suggestion
exclude = __init__.py, setup_package.py, conf.py, setup.py, version.py, conftest.py
```

---

Fixed